web-server/v0.4/src/pages/Dashboard/Summary.js

```diff
@@ -4,6 +4,8 @@ import { Select, Spin, Tag, Table, Button, Card, notification } from 'antd';
 import axios from 'axios';
```

Comment:
Can we remove this import since axios is not being referenced in this component?

Reply:
Yes, I'll remove that.